src/ontology/mondo-ingest.Makefile

```diff
@@ -187,6 +190,16 @@ config/term_exclusions.txt config/exclusion_reasons.robot.template.tsv: config/o
 	cat config/*_term_exclusions.txt > config/term_exclusions.txt; \
 	awk '(NR == 1) || (NR == 2) || (FNR > 2)' config/*_exclusion_reasons.robot.template.tsv > config/exclusion_reasons.robot.template.tsv
 
+# todo: Is it desired for there to be merged `.ttl` and `.obo` files? How to do that? Pass multiple `--input` and use the merged `config/exclusion_reasons.robot.template.tsv` for --template?
```

Comment:
@matentzn Do you want (1) simply the OBO and TTL exclusion reasons per ontology? Or do you also want (2) A merged `.ttl` and `.obo` file as well? 

If you also want (2), how to do that? Pass multiple ontology `.owl` files to `--input`, and for `--template` I use the single merged `config/exclusion_reasons.robot.template.tsv`?

Reply:
Nico wrote a merge command.

Reply:
remove OBO from list of things. all good.